src/module/home.js

```diff
@@ -0,0 +1,72 @@
+import { url, displayHome } from './variables.js';
+import { reservPop } from './reservations.js';
+import { createLike } from './likes.js';
+import { getLikes } from './likes.js';
+
+const likesCount = (target, likesArray, numOfLikes) => {
+  likesArray.forEach((obj) => {
+    if (obj.item_id === target.id) {
+      numOfLikes.innerHTML = `${obj.likes} likes `;
+    }
+  });
+};
+
+const pasteLike = async (item, paragraph) => {
+  if (paragraph === undefined) {
+    const paragraph = document.querySelectorAll('.like-show')
+    const getAllLikes = await getLikes();
+    getAllLikes.forEach((like) => {
+      if (like.item_id === item) {
+        paragraph.forEach((card) => {
+          if (card.id === item) {
+            card.innerHTML = `${like.likes}`
+          }
+        })
+      }
+    })
+  }
+ 
+}
+
+const display = async () => {
```

Comment:
- [ ] I kindly suggest that you follow the given mockup to create your home page and also add the items counter on the Home Page.
- [ ] I kindly suggest that you implement the pop up and don't append the data to the bottom of the page.

| Your Design  | Mock up Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/181872331-0c6545e2-c05b-408f-a5be-c397647451fc.png) | <img width="429" alt="mock" src="https://user-images.githubusercontent.com/52934128/181872347-51b182b9-fdd1-4682-a364-dda309a439d7.PNG"> |